sympy/core/containers.py

```diff
@@ -269,6 +269,14 @@ def _sorted_args(self):
         return tuple(sorted(self.args, key=default_sort_key))
 
 
+converter[dict] = lambda d: Dict(*d.items())
```

Comment:
How `converter` works? Is it something inside `SymPy`, like some class which have overrides `__setitem__` and `__getitem__`? 
PS - I am asking this out of curiosity. Thanks in advance.

Reply:
It's a dicstionary that can be imported where it is need (like in `sympify` that has a key that is a cls/type and value that is a unary function that converts the passed object into a SymPy object.